### PR TITLE
Add new singleCta variation of UniversalExpandedNavbar

### PR DESCRIPTION
**Pull request by coleEthos**
### Please review these reminders and either check the box or explain why not:

- [x] Read and followed the [contributing guidelines](https://eds.ethoslabs.io/#/Guidelines?id=section-contribute)?
- [x] Component is exported from [index.js](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.js)?
- [x] Type is exported from [index.d.ts](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.d.ts) so Typescript users can consume it? Added a test and ran `yarn test:types`?
- [x] Bumped the yarn version?

### Referencing PR or Branch (e.g. in CMS or monorepo):
- https://github.com/getethos/cms/pull/4868

### Screenshots and extra notes:
Screen Shot 2021-11-12 at 9 05 07 AM
Screen Shot 2021-11-12 at 9 05 03 AM

### CODE REVIEW

File: src/components/UniversalNavbarExpanded/MobileNav/MobileNav.module.scss [hunk 1]
```diff
@@ -51,6 +51,21 @@ $navbar-border: solid 1px var(--GrayStrokeAndDisabled--translucent);
...
+  &.singleCta {
+    :global {
+      @include for-phone-and-tablet {
+        .cta-button.show-when-scrolled.navbar-expanded {
+          right: 7px;
+        }
+      }
+      @media (max-width: $phone-range-small-end) {
+        .cta-button.show-when-scrolled.navbar-expanded {
+          right: 7px;
```
Comment (WesleyDRobinson) [inline src/components/UniversalNavbarExpanded/MobileNav/MobileNav.module.scss:64]: is this not covered by `for-phone-and-tablet`?

File: src/components/UniversalNavbarExpanded/SampleContent.js [hunk 2]
```diff
@@ -24,6 +24,10 @@ const playIcon = () => (
  * TODO when Kustomer implementation is ready and we want to include a talk button
```
Comment (WesleyDRobinson) [inline src/components/UniversalNavbarExpanded/SampleContent.js:24]: the Kustomer implementation has come and gone :|

File: src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js [hunk 3]
```diff
@@ -54,6 +55,7 @@ const UniversalNavbarExpanded = ({
   trackSecondaryCtaClick,
   links,
   estimateExperiment,
+  singleCta,
```
Comment (WesleyDRobinson) [inline src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js:58]:
```suggestion
  singleCta = {},
```
so you don't get any cannot read property href of undefined... errors

File: src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js [hunk 4]
```diff
@@ -39,6 +39,7 @@ import styles from './UniversalNavbarExpanded.module.scss'
  * @param {string} logoHref - Href for the logo
  * @param {object} links - URLs and text
  * @param {boolean} estimateExperiment - enable the estimate experiment button/copy
+ * @param {string} singleCta - A single CTA URL to link to in a reduced version of the navbar
```
Comment (WesleyDRobinson) [inline src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js:42]: Update to match the propTypes def of `singleCta = { href: string, title: string }`

File: src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.d.ts [hunk 5]
```diff
@@ -51,6 +51,7 @@ type UniversalNavbarExpandedProps = {
     NAVLINKS: NavLink[]
     /** Estimate copy experiment on optimizely */
     estimateExperiment?: boolean
+    singleCta?: string
```
Comment (WesleyDRobinson) [inline src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.d.ts:54]:
```suggestion
    singleCta?: { href: string; title: string }
```

linting may be off if you accept this change, idk 😄

File: src/components/UniversalNavbarExpanded/SampleContent.js (hunk 2 above)
Comment (coleEthos) [reply src/components/UniversalNavbarExpanded/SampleContent.js:24]: Ha, good catch, will remove this.

File: src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.d.ts (hunk 5 above)
Comment (coleEthos) [reply src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.d.ts:54]: Will take a look!

File: src/components/UniversalNavbarExpanded/MobileNav/MobileNav.module.scss (hunk 1 above)
Comment (coleEthos) [reply src/components/UniversalNavbarExpanded/MobileNav/MobileNav.module.scss:64]: I believe I added this to makeup for the hamburger nav not being shown in this variation, but will review locally again.

Comment (coleEthos) [reply src/components/UniversalNavbarExpanded/MobileNav/MobileNav.module.scss:64]: Ah yeah, not sure why I duplicated this here. Developer error, good catch.

File: src/components/UniversalNavbarExpanded/MobileNav/MobileNav.js [hunk 6]
```diff
@@ -78,6 +79,7 @@ const MobileNav = ({
   hideMobileCta,
   ctaButtonTrackingFunction,
   LinkComponent,
+  singleCta,
```
Comment (WesleyDRobinson) [inline src/components/UniversalNavbarExpanded/MobileNav/MobileNav.js:82]:
```suggestion
  singleCta = {},
```
 ah, better do it here too, sorry i missed this before

**Review by WesleyDRobinson — APPROVED** [review]
🧑‍🎤 🤙

**Pull request merged**